Update README.md to include examples, use cases etc

mjmayer opened the pull request.

mjmayer committed:
Update README.md to include examples, use cases etc

ruffrey commented on README.md:
```diff
@@ -1,10 +1,92 @@
+this application meet that Mailsac doesn't already provide?
+
+Mailsac has limitations on what can be viewed without signing up for an account. Only the latest email in a public
+mailbox can be viewed withing an account. Mail in a private domain cannot be viewed without signing in with an account
```
“withing”

ruffrey commented on README.md:
```diff
@@ -1,10 +1,92 @@
+
+## Use Cases
+
+[Mailsac](https://mailsac.com) already offers disposable email without a need to sign up for an account. What need does
```
Needs a header about Mailsac.com limitations or that this starts with “Background” info. I expected to read a useful feature after the header above, but it’s only limitations.

ruffrey commented on README.md:
```diff
@@ -1,10 +1,92 @@
+An instructor may want students, who are young in age and don't have an email address, to sign up for an account with
+a web service used in the class. The Mailsac Self-Hosted Temporary Email User Interface application provides a
+simplified interface for students to view email sent to a private mailsac hosted domain without the need to sign up for
+a mailsac account or email address.
```
I think just as important their users won’t have all the features of Mailsac UI, just view mail.

ruffrey commented on README.md:
```diff
@@ -1,10 +1,92 @@
+After a successful deployment you can click on the image of the application  to be taken to the live application.
+
+**NOTE** There is currently no authentication on this application. Anyone with the URL will be able to view emails
+and domains associated with the Mailsac API key that was used.
```
and the Ops will be tracked to your Mailsac account

mjmayer committed:
Update README with feedback

- Specify app allows read only access
- Call out that operations will be tracked in the account from which the
  API key is generated
- Fix mispellings
